apps/explore/tests.py

```diff
@@ -1,3 +1,2 @@
-from django.test import TestCase
```

Comment:
TestCase is considered a primary import for this type of file, even if it's not used.

Reply:
Change reverted.